### PR TITLE
Issue #5881: False positive for multiline imports

### PR DESCRIPTION
**Pull request by pbludov**
Issue #5881 

When the `import` token and the imported package name are on different lines, the first line should be compared.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java [hunk 1]
```diff
@@ -424,17 +425,15 @@ else if (option == ImportOrderOption.INFLOW) {
 
     /**
      * Shares processing...
-     *
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java]: Undo unnecessary change.

**Comment by romani** [comment]
Ok, just be sure, @pbludov please generate diff report on some config (looks like change affect all modes), to make sure to didn't miss anything.

**Comment by romani** [comment]
Let's do such update in separate issue, too much is moving in this Check now.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java (hunk 1 above)
Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java]: done

**Comment by romani** [comment]
@pbludov , please confirm that regression diff testing on some config does not have unexpected.
No exception regression is covered by wercker, but automated diff regression is still our dream, even we have some first step for this - https://github.com/checkstyle/regression-tool

**Comment by pbludov** [comment]
Regression report:
https://pbludov.github.io/issue-5881/
Zero difference.
